testLatestAkkaSnapshot.sh

```diff
@@ -0,0 +1,6 @@
+VERSION=$(curl https://repo.akka.io/snapshots/com/typesafe/akka/akka-actor_2.13/ | grep -o '2.6\.[0-9]*+[0-9]*-[0-9a-z]*' | sort -r | head -n 1)
+
+echo "Running with version Akka $VERSION"
+
+sbt -jvm-opts .jvmopts-travis -Doverride-akka.version=$VERSION test
```

Comment:
In https://github.com/akka/akka-persistence-cassandra/pull/664 this is called `"override.akka.version"` ?

Reply:
Doh, also found out that the sort doesn't work for all versions so fixed that too